src/oc/web/components/ui/section_editor.cljs

```diff
@@ -382,16 +391,20 @@
                                                (dismiss))}))}
               "Delete section"])
           [:button.mlb-reset.create-bt
-            {:on-click #(let [section-node (rum/ref-node s "section-name")
-                              inner-html (.-innerHTML section-node)
-                              section-name (utils/strip-HTML-tags inner-html)
-                              personal-note-node (rum/ref-node s "personal-note")
-                              personal-note (when personal-note-node (.-innerText personal-note-node))
-                              next-section-editing (merge section-editing {:slug utils/default-section-slug
-                                                                           :name section-name})]
-                          (dis/dispatch! [:input [:section-editing] next-section-editing])
-                          (when (fn? on-change)
-                            (on-change next-section-editing personal-note)))}
+            {:on-click (fn [_]
+                        (if (< (count @(::section-name s)) min-section-name-length)
```

Comment:
This logic should go into the section action so that it can be reused for other places that create the section.  The action should check the length then dispatch the error.